cross-reference/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim-buster
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Set work directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential mariadb-client libmariadbclient-dev
+
+# Install python dependencies
+COPY requirements.txt /app
+RUN pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt
+
+# Copy project
+COPY ./crossreference /app/
+
+# Add and run as non-root user
+RUN adduser --disabled-password --gecos '' cr
+USER cr
+
+# Run gunicorn
+CMD ["bash", "-c", "python manage.py collectstatic --noinput && gunicorn crossreference.wsgi:application --bind 1.0.0.0:25432"]
```

Comment:
yep, that's probably 0.0.0.0/0 (not sure if there is a best practice here, it maybe localhost too since the port is going to be exposed on localhost host by docker-compose).

Reply:
yep, that's a typo